packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -73,17 +188,25 @@ const BaseButton = ({
       }}
       disabled={isDisabled}
       activeColor={activeColor}
+      buttonHeight={buttonHeight}
```

Comment:
```suggestion
      height={buttonHeight}
```
how about just `height`? since button is already in context. ideally it shall be `minHeight`